### ISSUE TITLE
Fatal error on PHP 7 

### ISSUE THREAD

**Issue by docwilmot** [backdrop-contrib/bootstrap_lite#36]
Every page crashes immediately upon installing this theme, with error `Fatal error: Uncaught Error: Unsupported operand types in C:\wamp64\www\backdrop\core\includes\common.inc on line 3955`. This is resolved by adding an empty attributes key to any external css file added in `hook_css_alter()`, so example:
```
    $css[$cdn] = array(
      ...
      'weight' => -2,
      'attributes' => array(),
    );
```

**Comment by bugfolder** [comment]
Thanks for including the fix with the report!

I'd like to look into this a bit more deeply. The `attributes` key isn't documented in the description of [`backdrop_add_css()`](https://docs.backdropcms.org/api/backdrop/core%21includes%21common.inc/function/backdrop_add_css/1).

What version of Backdrop were you using? In the current version, 1.18.3, line 3955 of common.inc is a comment in the docblock. (It's the docblock of `backdrop_build_css_cache()`, so the true culpril line is probably nearby.)

**Comment by indigoxela** [comment]
Without the latest changes (BACKDROP-SA-CORE-2021-002) line 3955 is this one:

https://github.com/backdrop/backdrop/blob/9732bf36ee70c22175fd1d6255b472437434a601/core/includes/common.inc#L3961

`$element['#attributes'] += $item['attributes'];`

**Comment by indigoxela** [comment]
Hm. Could this be a negative side effect of https://github.com/backdrop/backdrop-issues/issues/4981? Might need more recherche. Then it might even be a core bug. (Merged but not released yet.)

**Comment by bugfolder** [comment]
Sure looks that way.

At the very least, the `attributes` tag needs to be documented in `backdrop_add_css()` (since it seems to be required), but also maybe it should be optional, so as not to break existing code.

**Comment by yorkshire-pudding** [comment]
@docwilmot @bugfolder  - it looks like this issue should be closed.  I haven't experienced this issue and the fix above looks like it should fix this.

**Comment by bugfolder** [comment]
Agreed. PR https://github.com/backdrop/backdrop/pull/3610, which addressed issue https://github.com/backdrop/backdrop-issues/issues/5070, was merged into Backdrop 1.19.0, which was released May 15, 2021. So this issue should no longer exist for up-to-date sites.
